lib/string-utils.ts

```diff
@@ -125,3 +125,21 @@ export function isInternalLink(link: string): boolean {
     domain === "wargabantuwarga.com"
   );
 }
+
+/**
+ * Mark up text with highlight tag
+ *
+ * @param {string} text original text
+ * @param {string} keyword string to be highlighted
+ * @returns {string} `true` if `value` is empty. `false` otherwise
+ */
+export function markText(text: string, keyword: string): string {
```

Comment:
Please write some test to ensure that this function works as expected so that we can make sure that future changes won't introduce regressions on this feature.

Reply:
Yes i just made unit test for this @zainfathoni